infra/gravity/node_commands.go

```diff
@@ -74,6 +74,12 @@ type Gravity interface {
 	Client() *ssh.Client
 	// Will log using extended info such as current tag, node info, etc
 	Logger() logrus.FieldLogger
+	// Connect connects the node to the cluster.
```

Comment:
Nit: Maybe it's just me, but I don't think of blocking / unblocking networking as Connect/Disconnect. Maybe Cordon/Uncordon or BlockNetwork/UnblockNetwork would be a bit better based on what I think these functions do.

Reply:
@knisbet @bernardjkim good point, in networking world the term is `Partition/Unpartition`